src/screens/FeedScreen/Content.js

```diff
@@ -255,6 +255,16 @@ const Content = ({
     });
   };
 
+  function topicsChipPosition() {
+    if (item.post_type === POST_TYPE_POLL) {
+      if (item.pollOptions.length > 3) {
+        return 'relative';
+      }
+      return 'absolute';
+    }
+    return 'absolute';
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `topicsChipPosition` has been introduced to determine the position of topic chips based on the post type and number of poll options. However, it seems like there might be a logical error in this function. If `item.post_type` is not equal to `POST_TYPE_POLL`, the function will always return 'absolute', regardless of the length of `item.pollOptions`. This could potentially lead to incorrect positioning of the topic chips for non-poll posts that have more than 3 options. Please ensure that this behavior is intended.
```diff
-   function topicsChipPosition() {
-     if (item.post_type === POST_TYPE_POLL) {
-       if (item.pollOptions.length > 3) {
-         return 'relative';
-       }
-       return 'absolute';
-     }
-     return 'absolute';
-   }
+   function topicsChipPosition() {
+     if (item.post_type === POST_TYPE_POLL && item.pollOptions.length > 3) {
+         return 'relative';
+     }
+     return 'absolute';
+   }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->